docs/intro/reusable-apps.txt

```diff
@@ -181,50 +181,48 @@ this. For a small app like polls, this process isn't too difficult.
    license. Just be aware that your licensing choice will affect who is able
    to use your code.
 
-#. Next we'll create a ``setup.py`` file which provides details about how to
-   build and install the app.  A full explanation of this file is beyond the
-   scope of this tutorial, but the `setuptools docs
-   <https://setuptools.readthedocs.io/en/latest/>`_ have a good
-   explanation. Create a file ``django-polls/setup.py`` with the following
-   contents:
+#. Next we'll create ``setup.cfg`` and ``setup.py`` files which provide details
+   about how to build and install the app. A full explanation of these files is
+   beyond the scope of this tutorial, but the `setuptools docs
+   <https://setuptools.readthedocs.io/en/latest/>`_ have a good explanation.
+   Create the files ``django-polls/setup.cfg`` and ``django-polls/setup.py``
+   with the following contents:
+
+   .. code-block:: ini
+        :caption: django-polls/setup.cfg
+
+        [metadata]
+        name = django-polls
+        version = 0.1
+        description = A Django app to conduct Web-based polls.
+        long_description = file: README.rst
+        url = https://www.example.com/
+        author = Your Name
+        author_email = yourname@example.com
+        license = BSD License  # example license
```

Comment:
~~This is not required if the trove classifier below is specific enough. Not sure whether we want to mention that...~~

I guess that would beyond the scope as mentioned in the preceding paragraph, and also there is no specific enough classifier for BSD licenses.

Reply:
Just to be a bore, based on my https://github.com/django/django/pull/12013#discussion_r342001037, let's make this `BSD-3-Clause` to set a good example. :angel:

Reply:
That works for me. Done in the latest revision.